.github/workflows/test.yml

```diff
@@ -20,16 +20,22 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install flake8 pytest numpy pandas cython tables
+        pip install wheel flake8 pytest numpy pandas cython tables
     - name: Lint with flake8
       run: |
         # stop the build if there are Python syntax errors or undefined names
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
         # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
-    - name: Build ms2pip
+    - name: Build and install ms2pip
       run: |
-        pip install --editable .
+        python setup.py bdist_wheel
+        pip install dist/ms2pip*.whl
     - name: Test with pytest
       run: |
         pytest
+    - uses: actions/upload-artifact@v2-preview
+      with:
+        name: wheels
+        path: dist/ms2pip*.whl
```

Comment:
What is the reasoning to upload a wheel for every commit?

Reply:
If the wheel has been built on the GitHub server anyway (which can take up to 10 minutes), we could as wel keep it if we want to install it somewhere for testing. GitHub Actions only stores most recent action artifacts. So as far as I know, there's no real downside to uploading the test wheels.